File: University/AlphaGonturusS1.cs
using System;
using System.IO;
public class Main
{
    public static void Main(string[] args)
    {
        try (public const  scanner =  "Inputs";)
        {
            var n = Convert.ToInt64(Console.ReadLine());
            var b = Convert.ToInt64(Console.ReadLine());
            var result = Convert.ToString(int.Parse(String.valueOf(n),10),b);
            Console.Write(result.ToUpper());
        }
    }
}

File: University/AlphaGonturusS1.php
<?php

class Main
{
    function __construct(){
    }
    public static function Main_1()
    {
        $local_this = new Main();
        return $local_this;
    }
    public static function main_1(&$args)
    {
        try (
        public $scanner = "Inputs";)
        {
            $n = (int)readline();
            $b = (int)readline();
            $result = strval(intval(strval($n),10),$b);
            echo $result.toUpperCase();
        }
    }
}
Main::main_1($argv);
